src/meta/header.rs

```diff
@@ -257,6 +257,28 @@ impl LayerAttributes {
     pub fn with_position(self, data_position: Vec2<i32>) -> Self {
         Self { layer_position: data_position, ..self }
     }
+
+    /// Set all common camera projection attributes at once.
+    pub fn with_camera_frustum(
```

Comment:
Nice!

Reply:
Thanks, that's the best I could do without implementing a bunch of matrix operations that would be used nowhere else :)

Reply:
Can be used like this: `with_camera_frustum(mat1, mat2, (fov_x, fov_y), near..far)`